modules/pipeline/providers/cache/interface.go

```diff
@@ -26,4 +26,7 @@ type Interface interface {
 	GetOrSetPipelineYmlFromContext(pipelineID uint64) (yml *pipelineyml.PipelineYml, err error)
 	GetOrSetPassedDataWhenCreateFromContext(pipelineYml *pipelineyml.PipelineYml, pipelineID uint64) (passedDataWhenCreate *action_info.PassedDataWhenCreate, err error)
 	ClearPipelineContextCaches(pipelineID uint64)
```

Comment:
make this `ClearPipelineContextCaches` method in an individual block. It's different with other methods.